scripts/postinstall.sh

```diff
@@ -6,3 +6,4 @@ cp node_modules/testpilot-metrics/testpilot-metrics.js src/vendor
 cp node_modules/kinto-http/dist/kinto-http.js src/sidebar/vendor
 cp node_modules/quill/dist/quill.js src/sidebar/vendor
 cp node_modules/quill/dist/quill.snow.css src/sidebar/vendor
+cp node_modules/jose-jwe-jws/dist/jose.js src/sidebar/vendor
```

Comment:
I asked myself the question but since we didn't use the minified version for the other ones I didn't. Also the file is served locally so I don't think 30K are that important is it?